js/_website/src/lib/templates/gradio/03_components/chatbot.svx

```diff
@@ -38,21 +38,42 @@ demo.launch()
 ### Behavior
 
 The data format accepted by the Chatbot is dictated by the `type` parameter.
-This parameter can take two values, `'tuples'` and `'messages'`.
-
-
-If `type` is `'tuples'`, then the data sent to/from the chatbot will be a list of tuples.
-The first element of each tuple is the user message and the second element is the bot's response.
-Each element can be a string (markdown/html is supported),
-a tuple (in which case the first element is a filepath that will be displayed in the chatbot),
-or a gradio component (see the Examples section for more details).
+This parameter can take two values, `'tuples'` and `'messages'`. 
+The `'tuples'` type is deprecated and will be removed in version 6.0 of Gradio.
 
+### Message format
 
 If the `type` is `'messages'`, then the data sent to/from the chatbot will be a list of dictionaries
 with `role` and `content` keys. This format is compliant with the format expected by most LLM APIs (HuggingChat, OpenAI, Claude).
-The `role` key is either `'user'` or `'`assistant'` and the `content` key can be a string (markdown/html supported),
-a `FileDataDict` (to represent a file that is displayed in the chatbot - documented below), or a gradio component.
+The `role` key is either `'user'` or `'`assistant'` and the `content` key can be one of the following:
 
+1. A string (markdown/html is also supported).
```

Comment:
@aliabd can you please help me with the formatting of this list?

<img width="994" alt="image" src="https://github.com/user-attachments/assets/352fd470-11ed-4016-a87a-92a5d7b2e35b">

Reply:
Yeah all of the bullet points seem to be missing. e.g. in the guide:

<img width="873" alt="image" src="https://github.com/user-attachments/assets/77ec3f8f-7bd4-4c4e-bcf3-3a05f1312f51">

Reply:
I use lists where i don't want them to have markers (on navbars and search results etc) so there was a style clash i didn't notice. Fixed now

Reply:
Thanks @aliabd !